README.md

```diff
@@ -729,7 +735,7 @@ $ git push -u origin HEAD
 
 If you want to check out the other default configs which ```git push``` can take, visit the documentation for Git at https://git-scm.com/docs/git-config#git-config-pushdefault
 
-If you do not want to change the git configuration, you can also use:
+If you do not want to change the configuration, you can also use:
```

Comment:
perhaps "the global configuration for Git" would be more clear?

Reply:
I think it gives the wrong view. We didn't use the `--global` option in the previous command. So, we can't say "*global* configuration". Why do you think "... to change the configuration... " isn;t clear enough?

Regardless, reading the context a little I guess it's confusing readers a lot by using inconsistent terms. I'll try to fix the confusion in a commit. Let's see how it goes.